src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx

```diff
@@ -9,22 +9,19 @@ export interface ViewViabilityTestModalProps {
   open: boolean;
   accession: Accession2;
   onClose: () => void;
-  selectedTest: ViabilityTest;
-  setNewViabilityTestOpened: React.Dispatch<React.SetStateAction<boolean>>;
-  setSelectedTest: React.Dispatch<React.SetStateAction<ViabilityTest | undefined>>;
```

Comment:
i moved controlling code into the parent since the child component here is not  aware of what's internal to parent